lib/src/view/design_main_strand.dart

```diff
@@ -997,18 +990,20 @@ Future<void> ask_for_assign_dna_sequence(
 
   String dna_sequence;
 
+  int m13_rotation = options.m13_rotation;
```

Comment:
nit: `m13_rotation` is only used inside of the body of the if statement, so I would declare it inside of the if statement.

Reply:
It's used again near the end of the function.